integration/env/common.go

```diff
@@ -64,45 +59,32 @@ func init() {
 		panic(fmt.Sprintf("env var %q must not be empty", EnvVarGithubBotToken))
 	}
 
-	testedVersion = os.Getenv(EnvVarTestedVersion)
-	if testedVersion == "" {
-		panic(fmt.Sprintf("env var '%s' must not be empty", EnvVarTestedVersion))
-	}
-
 	registryPullSecret = os.Getenv(EnvVarRegistryPullSecret)
 	if registryPullSecret == "" {
 		panic(fmt.Sprintf("env var '%s' must not be empty", EnvVarRegistryPullSecret))
 	}
 
 	testDir = os.Getenv(EnvVarTestDir)
 
-	params := &framework.VBVParams{
-		Component: component,
-		Provider:  provider,
-		Token:     githubToken,
-		VType:     TestedVersion(),
-	}
-	versionBundleVersion, err = framework.GetVersionBundleVersion(params)
-	if err != nil {
-		panic(err.Error())
-	}
-	// TODO there should be a not found error returned by the framework in such
-	// cases.
-	if VersionBundleVersion() == "" {
-		if strings.ToLower(TestedVersion()) == "wip" {
-			log.Println("WIP version bundle version not present, exiting.")
-			os.Exit(0)
+	{
+		vbs := service.NewVersionBundles()
+
+		if pathutil.Base(testDir) == "update" {
+			// For the update test we want to create previous
+			// version so we can upgrade from it.
+			versionBundleVersion = vbs[len(vbs)-2].Version
+		} else {
+			versionBundleVersion = vbs[len(vbs)-1].Version
 		}
-		panic("version bundle version  must not be empty")
+
+		os.Setenv(EnvVarVersionBundleVersion, VersionBundleVersion())
```

Comment:
What is using this variable?

Reply:
My hunch would be that this is some legacy crap with `e2e-harness` and eventually some old templates assuming this is just there somehow. When the tests work without this, I would just drop it as well.